.github/workflows/publish-build-layer-images.yml

```diff
@@ -0,0 +1,206 @@
+name: Publish build layer images
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_tag:
+        description: "Docker tag to be published"
+
+permissions:
+  packages: write
+  pull-requests: write
+  id-token: write
+  contents: read
```

Comment:
Do we need all these permissions? Naively it looks like we should just need contents read and the ability to access the `secrets.DOCKERHUB_RELEASER_ROLE` ?